### PR TITLE
fix outdated information about pushing tags

### PR DESCRIPTION
**Pull request by WesleyGoncalves**
Fix information about pushing only annotated tags

Github issue #1426

### CODE REVIEW

**Comment by WesleyGoncalves** [comment]
In this commit, I suggest an update about the issue #1426. 
What do you think about it?

File: book/02-git-basics/sections/tagging.asc [hunk 1]
```diff
@@ -220,7 +220,7 @@ Now, when someone else clones or pulls from your repository, they will get all y
 [NOTE]
 .`git push` pushes both types of tags
 ====
-Pushing tags using `git push <remote> --tags` does not distinguish between lightweight and annotated tags; there is no simple option that allows you to select just one type for pushing.
+Pushing tags using `git push <remote> --tags` does not distinguish between lightweight and annotated tags; However, if you use `git push <remote> --follow-tags` only annotated tags will be pushed to the remote.
```
Comment (ben) [inline book/02-git-basics/sections/tagging.asc:223]: Let's split this into two sentences:

```suggestion
`git push <remote> --tags` will push both lightweight and annotated tags.
There is currently no option to push only lightweight tags, but if you use `git push <remote> --follow-tags` only annotated tags will be pushed to the remote.
```

Comment (WesleyGoncalves) [reply book/02-git-basics/sections/tagging.asc:223]: Nice!

Comment (WesleyGoncalves) [reply book/02-git-basics/sections/tagging.asc:223]: Sorry, should I commit or you do it?

Comment (ben) [reply book/02-git-basics/sections/tagging.asc:223]: You go ahead. I want you to get `git blame` credit for the fix. 😁

**Pull request merged**